app/controllers/concerns/github_settings_mixin.rb

```diff
@@ -4,7 +4,7 @@ module GithubSettingsMixin
   def github_admin
     redirect_to oauth_client.auth_code.authorize_url(
       state: oauth_authenticity_token,
-      scope: "#{oauth_scope},admin:org",
+      scope: "#{oauth_scope},admin:org,repo",
```

Comment:
It'd be cool if this string was built from an array which could then use `join(',')`. Classes which include `GithubSettingsMixin` could override the array reader and append new scopes.